feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/OrchestratorFragment.kt

```diff
@@ -77,7 +77,7 @@ internal class OrchestratorFragment : Fragment(R.layout.fragment_orchestrator) {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         if (savedInstanceState != null) {
-            orchestratorVm.requestProcessed = savedInstanceState.getBoolean(KEY_REQUEST_PROCESSED)
+            orchestratorVm.isRequestProcessed = savedInstanceState.getBoolean(KEY_REQUEST_PROCESSED)
```

Comment:
Is this the only actual change in the file and all other changes are essentially just adding extra line breaks?

Reply:
Yes, that's right. The formatting was applied.